test/categorical_algebra/FinCats.jl

```diff
@@ -2,6 +2,7 @@ module TestFinCats
 using Test
 
 using Catlab, Catlab.Theories, Catlab.CategoricalAlgebra, Catlab.Graphs
+using Catlab.CategoricalAlgebra.FinSets: VarSet
```

Comment:
Looks like some accidental/leftover edits to this file here and in the commented lines below.